### DISCUSSION TITLE
Proposal to increase Community Pool Tax & decrease Inflation - July 2024

### DISCUSSION THREAD

**Discussion by lechenghiskhan** [akash-network/community#641] [Governance Proposals]
## Background and Context

The last inflation update was completed with Proposals [240](https://www.mintscan.io/akash/proposals/240) and [241](https://www.mintscan.io/akash/proposals/241) at the end of December 2023. 

As July 2024 draws to a close, the Akash GPU marketplace has nearly tripled in size, funded 4 quarters of public events including [Akash Accelerate](https://www.mintscan.io/akash/proposals/250), brought [Provider Incentives](https://www.mintscan.io/akash/proposals/246) to life, and continues to lead DePIN as the most advanced and capable Supercloud. 

As the network scales, it is more important than ever to maintain a robust funding rate for the Community Pool while managing token emissions against the relative macroeconomic backdrop.  

## Proposal

We are proposing a decrease in overall token emissions coupled with an increase in the Community Pool tax to ensure adequate funding with reduced emissions. The estimated calculations can be viewed [here](https://docs.google.com/spreadsheets/d/1N65YoGNTNr_eyC8F90NWSu1_z9EUSYAbiC51tHKuRDE/edit?gid=1773715512#gid=1773715512). 

1. Increase the Community Pool tax from 40% to 50% toensure it is properly funded for provider/tenant incentives and other community initiatives.  

2. Decrease in inflation minimum from 13% to 10% and inflation maximum from 20% to 15%. This reduction, when taken into account the relative strength of AKT (up over 40%)  since the end of 2023, will blunt the economic impact on validators and stakers created by the increased tax and decreased emissions. These changes will reserve future emissions for securing the network and and incentive programs in the future. 

Please note that the actual impact to the effective inflation and staking APY are purely estimates and will depend on network-wide staking behavior. 

The use of Community Pool proceeds will always be requested via governance proposal and determined by detailed discussions within SIG-Economics, Steering Committee, and associated Working Group meetings.

**Comment by KamuelBob** [comment]
Thanks @lechenghiskhan, love the idea and good to see energy going into this topic.

My comments:
- The proposal results in ~281.3m AKT by July 2025 (see below re: fixed calculations), instead of the originally planned ~271.8m AKT. Akash is still missing a clear narrative/definition of the tokenomics structure to get to the max supply as one of our community members recently highlighted. Can you comment on where we go into the future?
- It would be great to see a justification about why $69m adding to the community pool over the next year is required. If we keep the 40% tax,  APR will remain higher than inflation (a key metric for some) and community pool will still grow by $55m. e.g. what was the total community pool spend in the last year?

Proposal wording:
- It would be great if you could mention/confirm that this does not have any impact on the max supply in the proposal
- Could you please describe in the proposal how it impacts the original token supply estimates and why that is or isn't a problem
- Could you mention in the proposal what the effective APR would be at current staking rates? e.g. 13.6%
- Suggest a proof read, minor spelling and grammar

Calculation Sheet:
- A7 "inflatoin"
- B17, B35 The name says Inflation but you've given APR, these should be 20% and 15% respectively
- C38 should be "Estimated monthly inflation"
- C22 Why does the formula include E21 (estimated effective inflation rewards) in the calculation? Is this to account slightly for the per/block compound nature of the inflation?
- I've updated the calculations and we instead arrive at supply of 281.3m AKT. Contact me directly if you would like a copy.
- C8 I'm not sure that this has been calculated correctly, I get it closer to 13.6%.

Cheers,
Kam

**Reply by lechenghiskhan** [reply]
Thanks for your thorough review and reply @KamuelBob !

**Your comments**
1. As far as where we see Akash tokenomics deep into the future, there are many potential paths, but the first is what was originally laid out in our whitepaper and I must refer you there first. Where things could end up in practice will depend on a huge battery of factors. One of the biggest ones is network spend and we need more time to see how that ultimately shapes up. Crawl, walk, run. Don't skip steps.

2. Increasing the community tax AND decreasing inflation will have the net impact of lowering what the CP will grow by. Unchanged,  the Community Pool will grow faster. Over the last 12 months, roughly 370,000 AKT per month (on average) have been spent powering everything from events, bounties, and acquisitions, to provider incentives and "Akash Chat." Please look at the Akash public roadmap to glimpse what is to come. To see how Community Pool funds were spent, please review [this](https://github.com/akash-network/community/tree/main/community-pool-spend-reporting). 

**Everything else**
1. As is consistent with past inflation proposals, only parameters being changed are explicitly called out. Max supply will not be impacted. It is simply not common practice nor practical to list everything that isn't changing. 

2. The math shown in the spreadsheet displays the estimates based on projections. Actual numbers can vary because staking behavior cannot be accurately predicted. Please read the proposal carefully and in its entirety: 

> "**_Please note that the actual impact to the effective inflation and staking APY are purely estimates and will depend on network-wide staking behavior._**"

3. Minor edit suggestions have been made 👍. Thank you for catching and suggesting those!

**Calculations**
1. Your math is not correct. It is not possible to have a staking APR that is less than Max Inflation if the Actual Bonded rate is lower than the Goal Bonded if actual inflation is already maxed. I encourage you to review how the inflation module works [here](https://docs.cosmos.network/main/build/modules/mint#nextinflationrate). If you'd like to jump into how this all works, I highly encourage you to attend the next SIG-Economics meeting. I'll be there 😎.

Thank you again!

**Reply by KamuelBob** [reply]
Hey @lechenghiskhan, thanks for the quick reply.

I've dm'd you on discord to try to help fix the remaining errors in the calculation sheet.

Hypothetically, if the Community Tax was 100%, then if inflation was 15%, Staking APR would be 0% (because there's nothing left for stakers) so I'm not sure I agree with your assessment. And yes thanks, I know how the inflation module works, I helped refine it in 2018. Happy to explain at the next SIG-Economics meeting but hopefully it's all sorted before then anyway.

Cheers

**Reply by lechenghiskhan** [reply]
I'll take a look in discord. 

The formula for staking APR is this: Inflation / actual bonded ratio * (1-community pool tax).
So for Akash, it's currently 20% / ~55.3% * (1-40%). 

By not taking into account staking ratio, you're omitting a huge variable.

**Comment by NightfuryMD** [comment]
Agree with lowering staking percentage but STRONGLY disagree with increasing community tax. There is already a significant allocation for CP. At this time, there is not enough demand for Akash services to justify this high community tax. There is a ton of unused GPUs, and not enough demand. The CP tax has more than sufficient capital at this time to help with growth efforts. If and when there is a strong need for community spend, we can revisit this and community can vote on it alacarte. Skewing these numbers will strongly disincentivize long term stakers.

**Reply by KamuelBob** [reply]
@NightfuryMD definitely agree, it would reduce the community pool income by 20% by leaving it where it is but probably pay itself back in price action because the psychology of staking rewards being lower than inflation is pretty unpopular to the general community.

**Reply by lechenghiskhan** [reply]
Thanks @NightfuryMD for participating. 

The important thing to take into account here is preparing for the future as spend is never linear, but occurs in bursts.
As the network continues to mature and accelerate, the CP will be called on to do more than ever before. 
I encourage you look at the concerns around the CP running out in prior discussions and how they were mitigated with increased funding. 

Adjusting for actual spend reactively isn't necessarily a good thing, but I understand your point. 

There is another purpose to increasing the CP tax: redirecting emissions towards the CP, which has the added benefit of reducing inflation further. So in summary, reducing ONLY inflation will significantly reduce both staking APR and CP growth. By also increasing the community pool tax, we help offset that slowdown and ensure the pool remains very healthy.

**Reply by cosmonought** [reply]
@lechenghiskhan I followed and agree with most everything, except (1) "redirecting emissions towards the CP [...] has the added benefit of reducing inflation further," which is confusing me. Do you mean it reduces inflationary sell pressure on the token from validators and stakers? The tokens are still being minted, I think, so the inflation is the same, it's just the destination that is changing, right? I might be missing something. (2) the deltas for the changes in the document are given in percentages rather than basis points, so I believe the change, e.g., from 13% to 10% is a delta of ~23% rather than 3%, and likewise the change from 20% to 15% is a delta of 25% rather than 5%.

@NightfuryMD I appreciate your concerns. I think the issue (as you suggest) is whether the CP can be better used to drive demand for Akash services (e.g., by providing provider incentives, strengthening developer onboarding strategies, enabling more significant attendance/participation at Web2 and Web3 networking/marketing events, etc.) versus the current staking rewards—and I'm not sure that stakers are necessarily users of Akash's services, though they provide the network security, so there's a bit of a push-and-pull dynamic. My feeling is that if the growth lies in attracting customers and users of Akash's services, a well-funded AND well-managed CP is crucial, so overall I am in support of the changes put forward here because Akash has (imo) strived for transparency and excellent stewardship of resources. 

But you and @KamuelBob both touch upon the psychological effect of adjusting the issuance parameters to stakers (though as Cheng mentioned, it is incorrect to say that staking rewards will be lower than inflation, unless the max bond rate has been surpassed), and so I think the discussion would benefit from recalling that this proposal follows on from [Prop 241](https://www.mintscan.io/akash/proposals/241) at the start of January which raised min inflation from 8 to 13% and max from 15 to 20%. So overall this proposal is still an increase in inflation, but it moderates it down from the elevated rates that were needed to fund the CP for the year's many activities.

**Reply by NightfuryMD** [reply]
@lechenghiskhan, your articulation makes sense. Let’s get this proposal on chain. You have my vote.

**Reply by KamuelBob** [reply]
@cosmonought Agree with everything you said except for:
> it is incorrect to say that staking rewards will be lower than inflation, unless the max bond rate has been surpassed

As described in https://docs.cosmos.network/v0.46/modules/distribution/03_begin_block.html, the community tax (along with proposer rewards) is taken from the staking rewards, so in the event that (in current conditions of 55% bonded ratio and 20% inflation) community tax is above ~45%, the staking APR will be less than the inflation.
![image](https://github.com/user-attachments/assets/a813c268-1030-48b3-9458-7550d886a8db)

FYI Current proposer reward parameters:
![image](https://github.com/user-attachments/assets/c34d9039-dfc0-4e23-a79b-7c79360928b5)

@lechenghiskhan it would be good if you could please fix the remaining errors in the calculation sheet. It is currently ~$5m out. Given your assumptions, it should look something like this (I've already sent a copy of this, let me know if you didn't get it yet):
![image](https://github.com/user-attachments/assets/c5700869-84ed-499c-82eb-36bfc2200f1a)

I support dropping the inflation (and believe now is the right time to do it) but not raising the community tax.

**Reply by KamuelBob** [reply]
Or with the correction that @cosmonought suggested on the delta percentages which I agree with:
<img width="325" alt="image" src="https://github.com/user-attachments/assets/aef382bf-0c8c-4088-b4c9-c94f526b1b4b">

**Comment by ghost** [comment]
Hey there and thanks @lechenghiskhan for putting up this prop. 

As I've studied the AKT tokenomics thoroughly for a Cito Zone tokenomics report I'm super glad that the community is taking first steps to lower Akash's inflation.

Here are some thoughts: 

1) I fully agree that AKT inflation has to be reduced. With 20% it is amongst the highest inflation rates in the entire crypto space and inevitable leads to sell pressure. Rather than lowering it to max 15%, I would prefer to going even lower like max 10% or going back to the previous inflation level of min 8% max 13%. 
High inflation rates deter investors and sheds a negative light on a project. 

2) As inflation gets reduced, we should make sure that the staking APR stays above annual inflation. 
Diluting stakers does not make any sense, might cause a massive backlash from the community and basically disincentivizes stakers. Why should I lock up my tokens for 3 weeks and still get diluted by inflation. 
If we were to lower max inflation from 20% to 15% and increased the CP tax rate from 40% to 50%, assuming a constant staking participation rate of 55%, staking APR would be < annual inflation. => (15% / 55%) x 50% = 13.6% staking APR vs 15% annual inflation. This is a massive no go imo. 

3)  Community Pool funding: Keeping up a healthy CP should be of interest of every community to incentivize network participants, fund events, hackathons and other chain related projects and campaigns. However, overfunding a CP could also lead to various implications. I do understand that the Akash CP was underfunded and hence increasing inflation and the CP tax made a lot of sense. 
Right now though, I think the CP is decently funded with 8.78M $AKT (3.57% of circ. supply) or $27,6M. 

As @lechenghiskhan stated, over the past 12 months, an average of 370K AKT have been spent per month, making it 4.4M a year. Looking at these numbers we should consider that AKT trades way below $1 for almost the entirety of 2023, hence projecting a similar CP spent (4.4M AKT) for the years to come is a bit exaggerated. Thus I oppose increasing the community pool tax to 50%. 

It's very hard to argue how much CP funding will be needed to support the success of Akash network in the years to come. Given the current CP funds 8.78M $AKT or $27,6M + the projected inflows over the next year (21,2M AKT or $74M) if inflation was reduced to max 15%, as stated above by @lechenghiskhan, we would end up with a massively bloated CP. I mean $74M additional CP funding in one year, solely based on inflation, this is out of this world if you ask me. No CP spends $70M+ a year, not even those of tier one projects like Solana. 

To sum it up: 
- Yes to lowering inflation (could also be lowered even further to max 10% or min 8% max 13%)
- No to increasing the CP tax to 50%

=> I'm looking forward to the next economics sig call, as I think the community might consider an fundamental revamp of the AKT tokenomics. Like laying down a clear path towards reaching max supply, lowering inflation to a sustainable level, finding an appropriate way to fund the CP and incentivize the supply side of the Akash marketplace.

**Comment by NightfuryMD** [comment]
100% agree with @Seppmos 
Reduce inflation to min8-max13%. 
do not increase CP tax to 50%. 

If we need more time to think about the CP tax, we should atleast pass the inflation reduction proposal first to reduce the sell pressure. Sell pressure has been non stop!! Eventually, when we have a consensus on CP tax we can do it as a separate proposal. 

Thanks

**Comment by lechenghiskhan** [comment]
Thank you all for your passion on this subject and for participating.

Considering the feedback, the cleanest path forward is to decouple the inflation parameter adjustments from the Community Pool tax changes. 

In that light, I'd like to remove the CP tax increase from this proposal and only adjust the inflation parameters as follows:
- Inflation minimum: from 13% down to 8%
- Inflation maximum: from 20% down to 13%

We'll push this on-chain this week and reserve any changes to the CP for the future as the needs arise.

**Reply by NightfuryMD** [reply]
Great! Let’s push it on chain 👍

**Comment by hewi333** [comment]
Proposal is [live](https://www.mintscan.io/akash/proposals/265) on-chain!

**Reply by KamuelBob** [reply]
Awesome, voted yes! It would be great to see an updated spreadsheet with the expected emissions and CP value added to this discussion for delegators to be able to review and make an informed vote.

From my calculations, this proposal brings us almost perfectly back to the originally planned AKT supply at this point in time, just with a 13% inflation instead of the planned ~10% at this point in time, meaning we will start to creep above the planned supply towards the end of this year, which is a great result imo.

The funding to the community pool will however be reduced compared with the original planned wording of this proposal, so I support prompt attention on this should the available funds start to limit growth possibilities.

Thanks all for listening to the community!

**Reply by hewi333** [reply]
@KamuelBob [Here](https://docs.google.com/spreadsheets/d/1Xuo2H2We4m5YdUF0RqcyRxC5ujHp7ghsm0olbxJgDJo/edit?gid=1773715512#gid=1773715512) is an updated spreadsheet using numbers from today. Please let me know if you have any questions.

**Comment by SophoclesSummer** [comment]
This proposal was approved on 8/8/2024 by a 92% majority. 
https://www.mintscan.io/akash/proposals/265
